### ISSUE TITLE
Question about --flippingtest

### ISSUE THREAD

**Issue by CallMeNumber1** [mboboGO/DVBE#3]
When I applied --flippingtest, the shape of inputs and targets from validation set change to (batch_size, 2, 3, 448, 448) and (batch_size, 2) respectively.  And later the code fail to work, reporting error in the part of  accuracy computing, which bother me a lot. May i ask that have you tried --flippingtest in the released code and how  to fix it ?

**Comment by mboboGO** [comment]
Sorry for the confusing, I will check the code soon.

**Comment by Rayyoana** [comment]
When I applied --flippingtest, I also meet this problem. 
And the experiment result reproduced without fine-tuning on the CUB dataset was 48.7%. Which is 7.8%  lower than the result of 56.5% in your paper.
